packages/salesforcedx-vscode-core/src/util/configUtil.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ConfigAggregator, ConfigFile, ConfigValue } from '@salesforce/core';
+import * as path from 'path';
+import { isNullOrUndefined, isUndefined } from 'util';
+import { getRootWorkspacePath } from './index';
+
+export enum ConfigSource {
+  Local,
+  Global,
+  None
+}
+
+// This class should be reworked or removed once the ConfigAggregator correctly checks
+// local as well as global configs. It's also worth noting that ConfigAggregator, according
+// to its docs checks local, global and environment and, for our purposes, environment may
+// not be viable.
+
+// Given a configuration key, check the local then the global configs for the value and return
```

Comment:
I think these methods are clear enough that the commented explanations may not be necessary

Reply:
Sure, I've removed them with this PR.